testing_and_setup/compass/ocean/global_ocean/EC60to30/init/config_initial_state.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
 <config case="initial_state">
 
-	<get_file dest_path="initial_condition_database" file_name="PTemp.Jan_p3.filled.mpas100levs.160127.nc">
+	<get_file dest_path="initial_condition_database" file_name="PotentialTemperature.01.filled.60levels.PHC.151106.nc">
 		<mirror protocol="wget" url="http://oceans11.lanl.gov/mpas_data/initial_condition_database/"/>
 	</get_file>
 
```

Comment:
@mark-petersen, why is this file (in `global_ocean`, not `global_ocean_V1`) being modified?

Reply:
When changing to a jigsaw-created mesh version of EC60to30, all other parts of the design, like vertical layers, should remain the same unless we make a group decision.  With the EC60to30V3 we tried 100 layer and 80 layer simulations, and settled on 60.  Now that we have more experience, we could put that on a list of discussion items for E3SM V2, but not for this PR.

Reply:
Okay, that's fine.  I just wanted to be clear, since it seemed like the PR was focused on re-creating the V1 grids.